src/pages/WorkspaceSwitcherPage.js

```diff
@@ -0,0 +1,319 @@
+                    <View>
+                        <Text
+                            style={[styles.mt3, styles.label]}
+                            color={theme.textSupporting}
+                        >
+                            {translate('common.workspaces')}
+                        </Text>
+                    </View>
+                    <PressableWithFeedback
+                        role={CONST.ROLE.BUTTON}
+                        onPress={() => {
+                            App.createWorkspaceWithPolicyDraftAndNavigateToIt();
+                        }}
+                    >
+                        {({hovered}) => (
+                            <Icon
+                                src={Expensicons.Plus}
+                                width={12}
+                                height={12}
+                                additionalStyles={[styles.buttonDefaultBG, styles.borderRadiusNormal, styles.p2, hovered && styles.buttonHoveredBG]}
+                            />
+                        )}
+                    </PressableWithFeedback>
+                </View>
+
+                {usersWorkspaces.length > 0 ? (
+                    <OptionsSelector
+                        ref={inputCallbackRef}
+                        sections={[usersWorkspacesSectionData]}
+                        value={searchTerm}
+                        shouldShowTextInput={usersWorkspaces.length >= MINIMUM_WORKSPACES_TO_SHOW_SEARCH}
+                        onChangeText={onChangeText}
+                        selectedOptions={selectedOption ? [selectedOption] : []}
+                        onSelectRow={selectPolicy}
+                        shouldPreventDefaultFocusOnSelectRow
+                        highlightSelectedOptions
+                        shouldShowOptions
+                        autoFocus={false}
+                        disableFocusOptions
+                        canSelectMultipleOptions={false}
+                        shouldShowSubscript={false}
+                        showTitleTooltip={false}
+                        contentContainerStyles={[styles.pt0, styles.mt0]}
+                    />
+                ) : (
+                    <WorkspaceCardCreateAWorkspace />
+                )}
+            </>
+        ),
+        [
+            inputCallbackRef,
+            onChangeText,
+            searchTerm,
+            selectPolicy,
+            selectedOption,
+            styles.alignItemsEnd,
+            styles.borderRadiusNormal,
+            styles.buttonDefaultBG,
+            styles.buttonHoveredBG,
+            styles.flexRow,
+            styles.justifyContentBetween,
+            styles.label,
+            styles.mb1,
+            styles.mb3,
+            styles.mh4,
+            styles.mt0,
+            styles.mt2,
+            styles.mt3,
+            styles.p2,
+            styles.pt0,
```

Comment:
We can pass single `styles` instead of specifying that many styles.
```suggestion
            styles,
```